src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -273,17 +274,22 @@ void Engine::ApplyPerformanceHints(std::map<std::string, std::string> &config, c
 }
 
 void Engine::GetPerformanceStreams(Config& config, const std::shared_ptr<ngraph::Function>& ngraphFunc) {
-    const auto perf_hint_name = config.perfHintsConfig.ovPerfHint;
     // save hints parameters to model rt_info
     ov::AnyMap hints_props;
     std::string hint_name;
     const int latency_streams = get_num_numa_nodes();
     int streams;
+
     if (config.streamExecutorConfig._streams_changed) {
         streams = config.streamExecutorConfig._streams;
-    } else if (perf_hint_name == CONFIG_VALUE(LATENCY)) {
+        if (streams <= getAvailableNUMANodes().size()) {
```

Comment:
getAvailableNUMANodes() is old interface, now it is latency_streams.

Reply:
updated